tests/app/dao/test_users_dao.py

```diff
@@ -36,62 +38,55 @@ def test_create_user(notify_api, notify_db, notify_db_session):
     assert not user.platform_admin
 
 
-def test_get_all_users(notify_api, notify_db, notify_db_session, sample_user):
-    assert User.query.count() == 1
-    assert len(get_user_by_id()) == 1
-    email = "another.notify@digital.cabinet-office.gov.uk"
-    another_user = create_sample_user(notify_db,
-                                      notify_db_session,
-                                      email=email)
+@pytest.mark.usefixtures('notify_db_session')
```

Comment:
yeah that's a fair point - and for now, i imagine every almost single other usage of `notify_db_session` is when its being passed through to various fixtures, which will be staying for a while at least

could be more useful in admin, where we've got all of the stubbed out API calls, especially the login ones that we don't care about